database/sql/headers.go

```diff
@@ -34,6 +36,12 @@ const (
 	WHERE hash = ?
 	`
 
+	sqlHeaderFromHashAndState = `
+	SELECT hash, height, version, merkleroot, nonce, bits, chainwork, previous_block, timestamp, header_state, cumulated_work
```

Comment:
Are we using it somewhere else then for stop height,
if not, then I would change this to select only height. This way, it would probably use only an index without reaching for actual data.

Reply:
now we only take height